sdk/identity/Azure.Identity/src/AzureIdentityEventSource.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using Azure.Core;
+using Azure.Core.Diagnostics;
+
+namespace Azure.Identity
+{
+    [EventSource(Name = EventSourceName)]
+    internal sealed class AzureIdentityEventSource : EventSource
```

Comment:
Tests for event source?

Reply:
Do we have all the sources we write to listed/documented anywhere?

Reply:
Added tests for event source and exception model changes.